sw/device/lib/testing/flash_ctrl_testutils.c

```diff
@@ -0,0 +1,148 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/testing/flash_ctrl_testutils.h"
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include "sw/device/lib/base/mmio.h"
+#include "sw/device/lib/dif/dif_flash_ctrl.h"
+#include "sw/device/lib/runtime/hart.h"
+#include "sw/device/lib/testing/check.h"
+
+uint32_t flash_ctrl_testutils_wait_transaction_end(
```

Comment:
It looks like this only returns a bool (whether an error occurred during the operation). Perhaps the prototype needs to be updated? (for here and all its users)